CondFormats/EcalObjects/interface/EcalChannelStatusCode.h

```diff
@@ -17,25 +17,13 @@
 
 class EcalChannelStatusCode {
 public:
-  enum Code {
-    kOk = 0,
-    kDAC,
-    kNoLaser,
-    kNoisy,
-    kNNoisy,
-    kNNNoisy,
-    kNNNNoisy,
-    kNNNNNoisy,
-    kFixedG6,
-    kFixedG1,
-    kFixedG0,
-    kNonRespondingIsolated,
-    kDeadVFE,
-    kDeadFE,
-    kNoDataNoTP
-  };
+  typedef int Code;
+  static constexpr int kOk = 0, kDAC = 1, kNoLaser = 2, kNoisy = 3, kNNoisy = 4, kNNNoisy = 5, kNNNNoisy = 6,
+                       kNNNNNoisy = 7, kFixedG6 = 8, kFixedG1 = 9, kFixedG0 = 10, kNonRespondingIsolated = 11,
+                       kDeadVFE = 12, kDeadFE = 13, kNoDataNoTP = 14;
```

Comment:
Comparing to https://github.com/cms-sw/cmssw/pull/44487 this `enum` looks one that would be better kept as `enum` (#44487 showed only an equality comparison against a `float`(!))

Reply:
Hmm. Casting `kOk` to `double` would preserve the old behavior (IIUC). On the other hand, this is one of those places where equality comparison would be dangerous in general, and casting the `GetBinContent()` to `int` at least adds some rounding. Further OTOH, the equality comparison is done only against `0` (i.e. nothing has been added to the bin content), so I'd _think_ either way to work.